### ISSUE TITLE
Inline Documentation

### ISSUE THREAD

**Issue by orta** [robb/Underscore.m#38]
http://cocoadocs.org/docsets/Underscore.m/0.2.1/Classes/USArrayWrapper.html

It'd be cool to have some inline documentation for both Xcode & CocoaDocs :beer:

**Comment by robb** [comment]
Does CocoaDocs support TomDoc style comments? That's what I'm leaning towards, personally.

Either way, I'd love to automate the generation of [underscorem.org](http://underscorem.org), too!

Edit: Since Underscore.m is also somewhat of a hack in terms of syntax, I'm not sure AppleDoc will pick up the parameters correctly… :crying_cat_face:

**Comment by orta** [comment]
No, CocoaDocs ( and Xcode 5 ) don't support TomDoc. I imagine support could be added to appledoc, ( there's a thread about this on the ReactiveCococa repo. ) which would at least make it possible for CocoaDocs to work.

This idea of generating a site similar to underscorem.org is definitely something I could do with CocoaDocs ( say if you have under 10 classes, if the site is your design would you mind if I did something similar (but CocoaPods branded) ?

**Comment by robb** [comment]
Sure, go ahead.

When you say CocoaPods branded, do you mean mostly cocoa-colored? :chocolate_bar:

**Comment by orta** [comment]
Yeah, the colour schemes you see over at http://cocoapod.org - lots of browns & oranges. Don't worry, I'll find a way to make it tasteful, underscorem.org works from the beautiful contrasts so it'll be a thinker with a more constrained colour scheme.

**Comment by kvnify** [comment]
This is pretty old now. Where does this stand? Has something better come up in the last few years that might make this easier?  I'd love to get better inline docs in place for sure.

**Comment by orta** [comment]
[This file](https://github.com/robb/Underscore.m/blob/development/Underscore/Underscore%2BFunctional.h#L27-L93) needs [some](https://github.com/ashfurrow/Forgeries/blob/master/Pod/Classes/ForgeriesUserDefaults.h) `/// Docs` to get it inline in Xcode.

**Comment by kvnify** [comment]
Ok. I'll give the whole thing a one over before we update the pod. I'd love the docs to be there as well. The site is good, but not comprehensive.
